x-pack/legacy/plugins/siem/cypress/integration/lib/timeline/helpers.ts

```diff
@@ -19,6 +21,15 @@ import { DEFAULT_TIMEOUT } from '../util/helpers';
 export const toggleTimelineVisibility = () =>
   cy.get(TIMELINE_TOGGLE_BUTTON, { timeout: DEFAULT_TIMEOUT }).click();
 
+export const clearTimeline = () => {
+  cy.get(TIMELINE_DATA_PROVIDERS).then($provider => {
```

Comment:
[Per the discussion in 54475](https://github.com/elastic/kibana/pull/54475), consider replacing the implementation of this function such that is uses `Create New Timeline` to clear timeline state.